Practicals/prac_06/guitars.py

```diff
@@ -35,6 +35,4 @@ def main():
               f"${guitar.cost:10,.2f} {vintage_string}")
```

Comment:
sample out put from Lindsay is like---> print(f"Guitar {i}: {guitar.name:>20} ({guitar.year}), worth ${guitar.cost:10,.2f}{vintage_string}"), ,missing that part--->    Guitar {i}: {guitar.name:>20}